src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -212,6 +218,31 @@
      */
     public static final RegistryObject<Codec<NoneStructureModifier>> NONE_STRUCTURE_MODIFIER_TYPE = STRUCTURE_MODIFIER_SERIALIZERS.register("none", () -> Codec.unit(NoneStructureModifier.INSTANCE));
 
+    /**
+     * Stock holder set type that represents any/all values in a registry. Can be used in a holderset object with { "type": "forge:any" }
+     */
+    public static final RegistryObject<HolderSetType> ANY_HOLDER_SET = HOLDER_SET_TYPES.register("any", () -> AnyHolderSet::codec);
+   
+    /**
+     * Stock holder set type that represents an intersection of other holdersets. Can be used in a holderset object with { "type": "forge:and", "values": [list of holdersets] }
+     */
+    public static final RegistryObject<HolderSetType> AND_HOLDER_SET = HOLDER_SET_TYPES.register("and", () -> AndHolderSet::codec);
+
+    /**
+     * Stock holder set type that represents an intersection of other holdersets. Can be used in a holderset object with { "type": "forge:or", "values": [list of holdersets] }
```

Comment:
This is not an intersection but a union of other holdersets.
```suggestion
     * Stock holder set type that represents a union of other holdersets. Can be used in a holderset object with {@code { "type": "forge:or", "values": [list of holdersets] }}
```